pysatNASA/instruments/de2_vefimagb.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
+"""Module for the DE2 VEFI instrument.
```

Comment:
Repeated question, more directly placed. Why are there two DE2 VEFI instrument modules?

Reply:
See issue and pull referenced above.